Source/AH/Mixins/Throttling.lua

```diff
@@ -0,0 +1,50 @@
+AuctionatorAHThrottlingFrameMixin = {}
+
+local Mixin = AuctionatorAHThrottlingFrameMixin
+
+local THROTTLING_EVENTS = {
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_DROPPED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_QUEUED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_RESPONSE_RECEIVED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_SENT",
+  "AUCTION_HOUSE_THROTTLED_SPECIFIC_SEARCH_READY",
+  "AUCTION_HOUSE_THROTTLED_SYSTEM_READY",
+  "AUCTION_HOUSE_BROWSE_FAILURE"
+}
+
+function Mixin:OnLoad()
+  print("loaded")
+  self.ready = false
+  self.searchReady = false
+
+  self._lastCall = nil
```

Comment:
As noted above, lets not use `_` to indicate access modifier. If we really want this to be private in this scope, we can make it a local definition in this module?